scripts/release/homebrew/docker/formula_generate.py

```diff
@@ -126,6 +126,11 @@ def update_formula() -> str:
     upstream_sha = compute_sha256(HOMEBREW_UPSTREAM_URL)
     text = re.sub('sha256 ".*"', 'sha256 "{}"'.format(upstream_sha), text, 1)
     text = re.sub('.*revision.*\n', '', text, 1)  # remove revision for previous version if exists
+
+    # include pip when creating venv, see https://github.com/Azure/azure-cli/issues/27047
```

Comment:
It the link incorrect? https://github.com/Azure/azure-cli/issues/27047 doesn't seem to be related to `pip` in a venv.

Reply:
Update to homebrew pr link: https://github.com/Homebrew/brew/pull/15792